src/components/ebay-progress-bar-expressive/component.ts

```diff
@@ -0,0 +1,152 @@
+import { AttrString } from "marko/tags-html";
+import type { WithNormalizedProps } from "../../global";
+
+export const messageDurationStandard = 1500;
+export const messageDurationReducedMotionMultiplier = 1.5;
+export const messageFadeInDuration = 833;
```

Comment:
Do we want to allow users to modify these? Im fine leaving them as static variables though

Reply:
I don't think we should. We have a way for them to override `messageDurationStandard` on a per-message basis with the `duration` attribute, but I don't see a need for them to modify these constants.